core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSComfortType.java

```diff
@@ -1,7 +1,7 @@
 package fr.sncf.osrd.railjson.schema.rollingstock;
 
 public enum RJSComfortType {
```

Comment:
This struct should not be needed anymore. It has no difference with `RollingStock.Comfort` used everywhere else in the code. You should be able to remove it, remove the corresponding method `parseComfort`, and directly use `var comfort: RollingStock.Comfort? = null` in `STDCMRequest.kt`. There's a few modifications needed elsewhere afterwards :)